import-service/src/handlers/importFileParser.js

```diff
@@ -0,0 +1,42 @@
+const AWS = require('aws-sdk');
+const csvParser = require('csv-parser')
+
+const BUCKET = 'products-bucket-aws-epam';
+
+// const moveToParsedFolder = (event) => {
```

Comment:
why is it commented? any troubles?
btw, the `moveToParsedFolder` function should be `async`